daemon/graphdriver/driver.go

```diff
@@ -46,7 +48,6 @@ type ProtoDriver interface {
 	// String returns a string representation of this driver.
 	String() string
 	// Create creates a new, empty, filesystem layer with the
-	// specified id and parent. Parent may be "".
```

Comment:
its in experimental, we are hoping 1.10 in stable

On Thu, Oct 22, 2015 at 3:38 AM, Amos Folarin <notifications@github.com>
wrote:

> In daemon/graphdriver/driver.go
> <https://github.com/docker/docker/pull/12648#discussion_r42733039>:
>
> > @@ -46,7 +48,6 @@ type ProtoDriver interface {
> >  	// String returns a string representation of this driver.
> >  	String() string
> >  	// Create creates a new, empty, filesystem layer with the
> > -	// specified id and parent. Parent may be "".
>
> Super excited! Do we know if this is going into 1.9
> https://github.com/docker/docker/wiki/Engine-1.9.0 ?
> … <#1508f2153c6c6673_>
> On 12 October 2015 at 14:43, Phil Estes <notifications@github.com> wrote:
> In daemon/graphdriver/driver.go <#12648 (comment)
> <https://github.com/docker/docker/pull/12648#discussion_r41756844>>: > @@
> -46,7 +48,6 @@ type ProtoDriver interface { > // String returns a string
> representation of this driver. > String() string > // Create creates a new,
> empty, filesystem layer with the > - // specified id and parent. Parent may
> be "". Ugh--thanks for catching this @2opremio
> <https://github.com/2opremio> <https://github.com/2opremio> and thanks
> for quickly correcting it @HuKeping <https://github.com/HuKeping> <
> https://github.com/HuKeping> :) — Reply to this email directly or view it
> on GitHub <https://github.com/docker/docker/pull/12648/files#r41756844>.
> -- - Amos Folarin
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12648/files#r42733039>.
>